include/tvm/relay/op.h

```diff
@@ -81,6 +81,10 @@ class OpNode : public relay::ExprNode {
    *  This is in analogies to BLAS levels.
    */
   int32_t support_level = 10;
+  /*!
+   * \brief indicate whether output shape is data dependant.
+   */
+  bool shape_data_dependant = false;
```

Comment:
Can we simply make it an op attr (TShapeDataDependent), instead of making it a flag(that we need special setter function for).